plugins/hls-cabal-fmt-plugin/hls-cabal-fmt-plugin.cabal

```diff
@@ -0,0 +1,43 @@
+cabal-version:      2.4
+name:               hls-cabal-fmt-plugin
+version:            0.1.0.0
+synopsis:           TODO
+description:
+  Please see the README on GitHub at <https://github.com/haskell/haskell-language-server#readme>
+
+license:            Apache-2.0
+license-file:       LICENSE
+author:             The Haskell IDE Team
+copyright:          The Haskell IDE Team
+maintainer:         jana.chadt@nets.at
+category:           Development
+build-type:         Simple
+extra-source-files: LICENSE
+
+library
+  exposed-modules:  Ide.Plugin.CabalFmt
+  hs-source-dirs:   src
```

Comment:
```suggestion
  hs-source-dirs:   src
  ghc-options:      -Wall
```

Reply:
I think warnings would be useful, otherwise you don't see redundant imports, etc...

Reply:
Thank you, that is a good call.